CONTRIBUTING.md

```diff
@@ -20,13 +20,14 @@ following first, in order to get your bearings.
 If you just want to know how to contribute to the code, go to
 [Contribute to the code](#contribute-to-the-code).
 
-### Meet the community and the mainteners
+### Meet the community, the mainteners, and other Cucumber developers
 
 Smartbear is hosting a [community message board][cucumber-smartbear-community].
 This is a good place to meet users, the community, and to ask questions.
 
 You can also join the Cucumber Community Slack team:
 [register for an account][register-slack] then head over to [#intro][slack-intro].
+This is the place to be to meet the mainteners aka the core team.
```

Comment:
```suggestion
This is the place to be to meet the maintainers aka the core team.
```

Reply:
Thanks @olleolleolle 🙂

I'll fix that 👍

Reply:
I _really_ like what you are doing here. These texts are doing a big job.

Reply:
Thank you ☺️